### DISCUSSION TITLE
Json command from HomeAssistant

### DISCUSSION THREAD

**Discussion by Schoultzies** [awawa-dev/HyperHDR#894] [General]
There id a json command to switch on certain effects,  is there one to revert back to 'capture'? I would like an automation that 'captures' for video, and switches to an audio effect for music.

**Comment by Schoultzies** [comment]
Perhaps I could be more specific,  I am looking for a json command that replicates 'select source' in Remote Control.
Would be greatful for any assistance,  thanks.

**Comment by awawa-dev** [comment]
You need this:
https://github.com/awawa-dev/HyperHDR/blob/12ed1c3f70d43aa620dbb58c9727835016495726/www/js/remote.js#L228
=>
https://github.com/awawa-dev/HyperHDR/blob/12ed1c3f70d43aa620dbb58c9727835016495726/www/js/hyperhdr.js#L502
I never need it outside UI
